optax/_src/loss.py

```diff
@@ -461,6 +461,35 @@ def ctc_loss(logits: chex.Array,
   return per_seq_loss
 
 
+def convex_kl_divergence(log_predictions: chex.Array,
```

Comment:
Could you also add it to the public symbols in __init__.py and to the docs (see `kl_divergence` example)?

Reply:
Done.